### PR TITLE
[RFR] Update assignee in Polarian doc block for snapshot tests

### PR DESCRIPTION
**Pull request by prichard77**
## Purpose or Intent

- __Updating tests__ for feature snapshot to take ownership of them.

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by prichard77** [comment]
This PR is only changes to asignee in Polarion docblock and assignee_id in requirements.py. I am looking into why PRT failed, but should not be releated to this PR.

File: cfme/tests/infrastructure/test_snapshot.py [hunk 1]
```diff
@@ -100,6 +100,7 @@ def test_memory_checkbox(small_test_vm, provider, soft_assert):
 @pytest.mark.rhv1
 @pytest.mark.rhv3
 @test_requirements.rhev
+@test_requirements.snapshot
 @pytest.mark.meta(automates=[1571291, 1608475])
```
Comment (ganeshhubale) [inline cfme/tests/infrastructure/test_snapshot.py]: @prichard77 
```test_requirements.snapshot``` is already declared globally. So we don't need to add it to each test case from this file.

Comment (prichard77) [reply cfme/tests/infrastructure/test_snapshot.py]: Removed

**Pull request merged**